docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,44 @@
+# How to upgrade bootstrap kubelet
+
+## Introduction
+
+Bootstrap kubelet is the kubelet running on the node as a systemd service. Unlike most components on the cluster, lokoctl cannot update it. So there are some manual steps involved in updating the bootstrap kubelet.
```

Comment:
I would improve wording here, but I cannot come up with some specific suggestion.

`running on the node as a systemd service` sounds a bit weird to me. I'd expect something like `running outside Kubernetes cluster` or something.

`Unlike most components on the cluster` maybe we can change to `Unlike most cluster components`.

`lokoctl cannot update it`: I would mention, that this will change in the future. 

`So there are some manual steps involved in updating the bootstrap kubelet.` should we start sentences in docs from `So`? Maybe we rephrase to something like `Updating bootstrap kubelet requires manual steps` or something?